repose-aggregator/components/filters/saml-policy-translation-filter/src/test/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilterTest.scala

```diff
@@ -754,37 +1147,37 @@ class SamlPolicyTranslationFilterTest extends FunSpec with BeforeAndAfterEach wi
     }
 
     it("should initialize the cache when given a config") {
-      ReflectionTestUtils.getField(filter, "cache") shouldBe null
+      ReflectionTestUtils.getField(filter, "org$openrepose$filters$samlpolicy$SamlPolicyTranslationFilter$$policyCache") shouldBe null
```

Comment:
I agree with wanting to avoid going back to the Java branching on this one.  Also because it would be 1. for test code 2. that's using reflection.

I wonder if there's a more appropriate reflection library that'll work better with Scala classes.